### ISSUE TITLE
Some Xorg windows have no frame.

### ISSUE THREAD

**Issue by Thesola10** [papyros/papyros-shell#230]
Some windows supposedly rendered by X.org do not have a frame in Papyros and float over the multitask view without being listed by it. (apparently they are not handled by QtComposer) Please, fix this!

Windows affected
- LXDE apps
- Tomahawk
- Spotify
- Some other GTK apps

PS: Please provide a way to change keyboard layout too.
##

**Comment by jalona** [comment]
@Thesola10 select Papyros in dm and start?

**Comment by Thesola10** [comment]
No. What I mean is that, AFTER having started Papyros, on the desktop, some apps I launch do not have a frame and do not appear in the multitask view or in the taskbar.

**Comment by Thesola10** [comment]
I think I figured out part of the problem: Papyros does not frame Xwayland windows.
